### PR TITLE
docs: fix docs for new contract instantiation

### PR DESCRIPTION
**Pull request by vnepveu**
Some things I forgot during refactoring

### CODE REVIEW

File: fuels-abigen-macro/README.md [hunk 1]
```diff
@@ -40,7 +40,8 @@ fn compile_bindings_from_inline_contract() {
         "#
     );
 
-    let contract_instance = SimpleContract::new();
+    let dummy_client = FuelClient::new("").unwrap();
```
Comment (adlerjohn) [inline fuels-abigen-macro/README.md:43]: The string here is supposed to be the URL? Instead of a string with a magic value, could we use an `Option`?

Comment (digorithm) [reply fuels-abigen-macro/README.md:43]: Yeah, `""` is just too... ugly. Passing a `None` would make more sense, even though very few times a user will pass a `None` (or `""`) to this method.

Comment (adlerjohn) [reply fuels-abigen-macro/README.md:43]: Changing this can be left to a future PR, since it's a functional change.

**Review by adlerjohn — APPROVED** [review]

**Review by digorithm — APPROVED** [review]

File: fuels-abigen-macro/README.md (hunk 1 above)
Comment (adlerjohn) [reply fuels-abigen-macro/README.md:43]: @vnepveu can you file an issue?

Comment (digorithm) [reply fuels-abigen-macro/README.md:43]: Hmm the problem is that this is on the `fuel-core` side https://github.com/FuelLabs/fuel-core/blob/master/fuel-client/src/client.rs#L52.

Comment (adlerjohn) [reply fuels-abigen-macro/README.md:43]: Ah, then we can file an issue in `fuel-core`!

Comment (vnepveu) [reply fuels-abigen-macro/README.md:43]: Yes filing the issue. Should I update the `""` bit to a random non-empty string?

Comment (adlerjohn) [reply fuels-abigen-macro/README.md:43]: Well, no, since it's a URL. If the correct thing _currently_ is `""`, then the docs should say `""`.

**Pull request merged**